src/renderer/pages/project/script_editor/beat_selector.vue

```
@@ -33,6 +34,16 @@
 const emit = defineEmits(["emitBeat"]);
 const selectedBeat = ref(0);
 
+onMounted(() => {
+
+  if (props.currentBeatType) {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)